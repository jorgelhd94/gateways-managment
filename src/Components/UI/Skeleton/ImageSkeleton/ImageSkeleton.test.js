import React from 'react';
import renderer from 'react-test-renderer';
import ImageSkeleton from './ImageSkeleton';

test('ImageSkeleton render correctly', () => {
  const component = renderer.create(<ImageSkeleton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
