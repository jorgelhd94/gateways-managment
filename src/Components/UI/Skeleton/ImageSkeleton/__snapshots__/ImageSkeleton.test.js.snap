// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageSkeleton render correctly 1`] = `
<div
  className="bg-white w-full mx-auto rounded-2xl mt-8"
>
  <div
    className=""
  >
    <div
      className="grid grid-cols-1 mt-2"
    >
      <div
        className="h-24 bg-gray-200 rounded animate-pulse"
      />
    </div>
  </div>
</div>
`;
